### ISSUE TITLE
Add ariaControls input to SortingComponent instances

### ISSUE THREAD

**Issue by BenjaminEmiliani** [SAP/spartacus#15401]
Recent change (#14997) added a directive to handle aria-label and aria-controls for ng-select to comply with SAP AMP. `<cx-sorting>` element uses ng-select and we need to pass a new input ariaControls to the sorting component selector.

**Comment by KateChuen** [comment]
Tested on develop branch using yarn. 
- tested sorting on plp
- no AMP violation when sorting on plp
<img width="1199" alt="Screen Shot 2022-05-12 at 9 30 02 AM" src="https://user-images.githubusercontent.com/45409515/168131435-d237ed70-634e-4a95-ad5c-a02a5518ae63.png">
<img width="1109" alt="Screen Shot 2022-05-12 at 9 29 28 AM" src="https://user-images.githubusercontent.com/45409515/168131443-bb7146ce-aa96-46fa-a3db-60b8f5257099.png">
<img width="1061" alt="Screen Shot 2022-05-12 at 9 25 11 AM" src="https://user-images.githubusercontent.com/45409515/168131445-7f7e75ec-89a4-4dce-9ea2-4c62d8b45132.png">
<img width="1058" alt="Screen Shot 2022-05-12 at 9 24 48 AM" src="https://user-images.githubusercontent.com/45409515/168131447-f35ce812-3f5a-4913-b3c3-9b978d81db2b.png">
<img width="1076" alt="Screen Shot 2022-05-12 at 9 24 26 AM" src="https://user-images.githubusercontent.com/45409515/168131450-fdbfe076-fb2a-4223-b1cd-d7abcc0d8bd2.png">
<img width="1076" alt="Screen Shot 2022-05-12 at 9 24 04 AM" src="https://user-images.githubusercontent.com/45409515/168131453-0c8c020f-336c-4b2b-9fe9-d972a24868de.png">
<img width="1138" alt="Screen Shot 2022-05-12 at 9 23 21 AM" src="https://user-images.githubusercontent.com/45409515/168131456-6c4dcee1-647f-4a3c-bf8f-c26ae51781b8.png">
<img width="611" alt="Screen Shot 2022-05-12 at 9 22 42 AM" src="https://user-images.githubusercontent.com/45409515/168131461-686b47c6-be04-4f5b-b426-be000a254999.png">
